substrate-node/pallets/pallet-smart-contract/src/tests.rs

```diff
@@ -4537,3 +4581,25 @@ fn get_service_contract() -> types::ServiceContract {
 fn get_timestamp_in_seconds_for_block(block_number: u64) -> u64 {
     1628082000 + (6 * block_number)
 }
+
+fn activate_billing_accounts(had_solution_provider: bool) {
+    let pricing_policy = TfgridModule::pricing_policies(1).unwrap();
+    let foundation_account = pricing_policy.foundation_account;
+    let sales_account = pricing_policy.certified_sales_account;
+    let staking_pool_account = get_staking_pool_account();
+    let mut billing_accounts = vec![foundation_account, sales_account, staking_pool_account];
+    // push solution_provider_account if SmartContractModule::solution_providers(1) has Some
+    if had_solution_provider {
+        let solution_provider_account = SmartContractModule::solution_providers(1)
+            .unwrap()
+            .providers[0]
+            .who
+            .clone();
+        billing_accounts.push(solution_provider_account);
+    }
+    for account in billing_accounts {
+        if Balances::free_balance(&account) == 0 {
+            let _ = Balances::deposit_creating(&account, 500);
```

Comment:
We could add:
~~~rust
const CREATION_BALANCE: u64 = 500
~~~
and use it in all test file